curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-functional-programming-by-building-a-spreadsheet/6434759f78ec812264ff8f34.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-16
 
 # --description--
 
-In your callback, call `createLabel()` and pass `number` as the argument. You should see some numbers appear in your spreadsheet.
+In your callback, make two function calls. Call `createLabel()` and pass `number` as the argument. You should see some numbers appear in your spreadsheet.
```

Comment:
I would revert this sentence back to the way it was because we are not really making two function calls here. They only need to call the `createLabel` function

Reply:
`createLabel(number)` and `letters.forEach(...)` are two function calls.

Reply:
It could be better worded to this then

```suggestion
In your callback, you will need to make two function calls. Start by calling `createLabel()` and pass `number` as the argument. You should see some numbers appear in your spreadsheet.
```